src/main/java/com/puppycrawl/tools/checkstyle/api/AbstractLoader.java

```diff
@@ -49,12 +49,6 @@
  */
 public abstract class AbstractLoader
     extends DefaultHandler {
-    /** Feature that enables loading external DTD when loading XML files. */
-    private static final String LOAD_EXTERNAL_DTD =
-        "http://apache.org/xml/features/nonvalidating/load-external-dtd";
```

Comment:
@Nimfadora ,
Unfortunately, this is weird feature request from googlers , please see [commit](https://github.com/checkstyle/checkstyle/commit/636ae05d053e1a2ff59c1060a2736f22ab0f6c1b) and related issue #3605 .
How can we suppress pitest there or any other workaround (I am ok to move that code to special method `addFeauresForVerySecureJavaInstallations` and suppress pitest on this method)?